updating/updating-cluster-cli.adoc

```diff
@@ -10,7 +10,8 @@ OpenShift CLI (`oc`).
 
 [IMPORTANT]
 ====
-You can update your cluster to only a different minor version by using `oc`.
+You can update your cluster to only a patch version within your current minor
+version by using `oc`.
```

Comment:
Without `--force`, you have the same choices as in the web UI, and you can see those choices with `oc adm upgrade`.  From `oc adm upgrade --help`:

> If no arguments are passed the command will retrieve the current version info and display whether an upgrade is in progress or whether any errors might prevent an upgrade, as well as show the suggested updates available to the cluster.

Reply:
Did Clayton change his mind about the web UI being the best user experience for updates between versions? I thought that you couldn't change the channel.

Reply:
> Did Clayton change his mind about the web UI being the best user experience for updates between versions?

I doubt it ;).

> I thought that you couldn't change the channel.

You can `oc patch ...` like the examples [here][1].

[1]: https://github.com/openshift/openshift-docs/pull/16678#discussion_r326876856

Reply:
The UI is the best user experience because the cli doesn't have the tools to list/show you acceptable versions (maybe we should fix that)?

Reply:
I think that might be a good RFE. Along with the ability to easily retrieve and update the channels in the graph.

Reply:
> The UI is the best user experience because the cli doesn't have the tools to list/show you acceptable versions...

`oc adm upgrade` should list recommend targets from the graph (via ClusterVersion.status`).

Reply:
> [kalexand@localhost ~]$ oc adm upgrade
Cluster version is 4.1.20

> No updates available. You may force an upgrade to a specific release image, but doing so may not be supported and result in downtime or data loss.

When is update going to be working as intended?

Reply:
> When is update going to be working as intended?

No recommended upgrades from 4.1.20 is appropriate now.  What targets where you expecting to see?  You can test the `oc` functionality by running against 4.1.18, which should show 4.1.20 if you are in stable-4.1 or prerelease-4.1.

Reply:
Ah, then we're still stuck because the question's about updating between minor versions not within a minor version.